sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -95,7 +95,7 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
       mirrors.distinct :+ "https://archive.apache.org/dist" :+ PROCESS_TABLES.releaseMirror
     logInfo(s"Trying to download Spark $version from $sites")
     for (site <- sites) {
-      val filename = s"spark-$version-bin-hadoop3.tgz"
+      val filename = s"spark-$version-bin-hadoop3-scala2.13.tgz"
```

Comment:
We can change it back to `*-hadoop3.tgz` at Apache Spark 4.1 after the official Apache Spark 4.0 release.